Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapAreaSelectorView.swift

```diff
@@ -237,34 +226,36 @@ struct OnDemandMapAreaSelectorView: View {
         }
         
         var body: some View {
-            ZStack {
-                HandleShape(orientation: orientation, position: position, cornerRadius: OnDemandMapAreaSelectorView.cornerRadius)
-                    .stroke(.ultraThickMaterial, style: StrokeStyle(lineWidth: 5, lineCap: .round))
-                Color.clear
-                    .contentShape(Rectangle())
-                    .frame(width: 44, height: 44)
-                    .position(position)
-                    .gesture(DragGesture(coordinateSpace: .local)
-                        .updating($gestureState) { value, state, _ in
-                            switch state {
-                            case .started:
-                                state = .changed
-                                UISelectionFeedbackGenerator().selectionChanged()
-                            case .changed:
-                                resize(orientation, value.location)
-                            }
+            HandleShape(
+                orientation: orientation,
+                cornerRadius: OnDemandMapAreaSelectorView.cornerRadius
+            )
+            .stroke(.ultraThickMaterial, style: StrokeStyle(lineWidth: 5, lineCap: .round))
```

Comment:
```suggestion
#if os(visionOS)
            .stroke(.ultraThinMaterial, style: StrokeStyle(lineWidth: 5, lineCap: .round))
#else
            .stroke(.ultraThickMaterial, style: StrokeStyle(lineWidth: 5, lineCap: .round))
            .environment(\.colorScheme, .light)
#endif
```
Adjust stroke color on visionOS to `ultraThinMaterial` for a lighter appearance:
<img width="623" alt="Screenshot 2025-02-12 at 11 41 06 AM" src="https://github.com/user-attachments/assets/29ae335b-cc51-4001-94ce-d81b49312950" />